src/common/dataHooks/useMedia.js

```diff
@@ -61,7 +68,11 @@ export function useVideoObject({ id }) {
   return response?.data
 }
 
-export function useMediaUsageDetails({ id, docType }) {
+export function useMediaUsageDetails({
+  id,
+  docType,
```

Comment:
There's a ticket for refactoring it in the backlog, but just for reference, we are trying to move away from the use of the terms `document` and `docType` as they are legacy terms from nuxeo. In this instance `type` or `mediaType`  would be preferred.